.github/workflows/dispatch-build-bottle.yml

```diff
@@ -73,10 +73,9 @@ jobs:
         run: |
           if [ "$RUNNER_OS" = 'macOS' ]; then
             echo 'PATH=/opt/homebrew/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin' >> $GITHUB_ENV
-            echo 'GITHUB_ACTIONS_HOMEBREW_MACOS_SELF_HOSTED=1' >> $GITHUB_ENV
-          else
-            # No PATH needed on Linux as set by Docker
-            echo 'HOMEBREW_FORCE_HOMEBREW_ON_LINUX=1' >> $GITHUB_ENV
+          fi
+          if [ "${{matrix.runner != 'ubuntu-latest'}}" = "true" ]; then
```

Comment:
Complete pedantry but I wonder if we want to handle the case where we might ever use `macOS-latest` or pinned Ubuntu versions here? Ideally there'd be a way of actually detecting this not based on name 🤔. Don't block this PR on that, though.

Reply:
It could easily be done based on label and runner os.